pycodestyle.py

```diff
@@ -512,6 +513,32 @@ def missing_whitespace_after_import_keyword(logical_line):
             yield pos, "E275 missing whitespace after keyword"
 
 
+@register_check
+def missing_whitespace_after_match_case(logical_line):
```

Comment:
I would prefer if the existing checks were extended (this will provide better compatibility and performance)

Reply:
That was my initial idea. However, at least AFAIK it isn't that simple. As example, the best place would probably have been the `whitespace_around_keywords` check. The issue there is that it also checks for whitespace before the keyword and that `match` and `case` are only soft keywords.

It might make sense to rename the new check to something like `whitespace_around_soft_keyword`. That would at least provide options to extend it in the future.

https://github.com/PyCQA/pycodestyle/blob/61138ca629242a998b495ce05b5cd25df2429900/pycodestyle.py#L473-L494

Reply:
I don't see why you can't add it to the bottom of the code you've linked?  renaming or adding functions leads to a bunch of annoying issues created by operating system distributors that intentionally and erroneously violate dependency constraints so I'd really like to avoid adding / renaming functions when possible

Reply:
Ok, done.